### DISCUSSION TITLE
Does activating the toggles in Troubleshooting disable or enable detection methods?

### DISCUSSION THREAD

**Discussion by JaredRichardWilliam** [xanderfrangos/twinkle-tray#640] [Q&A]
Does this mean that the detection methods are enabled or disabled? The description states that this section allows me to disable them, not enable them.

![image](https://github.com/xanderfrangos/twinkle-tray/assets/113200291/585ef4be-2834-4d5c-858b-e546aebd31f0)

**Comment by xanderfrangos** [comment]
Turning on the toggle disables the detection method. So the screenshot show all 3 detection methods being disabled.
